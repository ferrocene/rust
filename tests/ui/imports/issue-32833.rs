//@ revisions: edition2015 edition2018
//@ [edition2015] edition: 2015
//@ [edition2018] edition: 2018..

use bar::Foo; //~ ERROR unresolved import `bar::Foo` [E0432]
              //~^ NOTE no `Foo` in `bar`
mod bar {
    use Foo; //[edition2018]~ ERROR unresolved import `Foo`
             //[edition2018]~^ NOTE no external crate `Foo`
}

fn main() {}
