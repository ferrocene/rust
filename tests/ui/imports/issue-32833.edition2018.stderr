error[E0432]: unresolved import `bar::Foo`
  --> $DIR/issue-32833.rs:5:5
   |
LL | use bar::Foo;
   |     ^^^^^^^^ no `Foo` in `bar`

error[E0432]: unresolved import `Foo`
  --> $DIR/issue-32833.rs:8:9
   |
LL |     use Foo;
   |         ^^^ no external crate `Foo`
   |
help: consider importing this unresolved item through its public re-export instead
   |
LL |     use crate::Foo;
   |         +++++++

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0432`.
